JobConfig/primary/PiPlusCalib_bfgeom_DS70.fcl

```diff
@@ -0,0 +1,15 @@
+#
+# generate and produce Detector Steps from flat spectrum using mu+ target stops
+#
+# original author: S Middleton
```

Comment:
Maybe you could change the description here to be a bit more complete and change the author

Reply:
done